### DISCUSSION TITLE
Nuxt 3 i18n localePath not translating the url

### DISCUSSION THREAD

**Discussion by digitalforgeangelo** [nuxt/nuxt#20532] [Questions]
I am having various problems with the routing system in Nuxt 3. 

1. When I visit my website (/) it does not automatically put the language prefix while it should do that. I am using the "prefix" strategy.

2. I have a unique URL for each page by using this code:

```
<script setup>
defineI18nRoute({
  paths: {
    nl: '/blogs/my-blog',
    de: '/bloggt/my-blog',
    pl: '/blogi/my-blog',
    tr: '/bloglar/my-blog',
  }
})
</script>
```

but when I make a link to the route, it always stays in the "nl" language and it does not change.

```
<NuxtLink :to="localePath('/blogs/my-blog')" :title="$t('blogs_1_title')">
<span class="absolute inset-0"></span>
    {{ $t('blogs_1_title') }}
</NuxtLink>
```

I do see the prefix change i.e. nl, de, pl or tr but not the "text" of the url itself, in this case the word "blogs".

When I check my routes with the Vue browser plugin, I do see the correct routes!

![image](https://user-images.githubusercontent.com/85583885/234647172-6085b3de-ba8a-49b4-9b66-2dc240f29da0.png)

Can anyone please tell me what I am doing wrong? Been browsing the documentation but I am unable to figure out what I am doing wrong.

**Comment by warflash** [comment]
May be worth asking this over on the i18n module itself https://github.com/nuxt-modules/i18n
